src/Plugin/formatter/FormatterJson.php

```diff
@@ -98,13 +117,17 @@ protected function extractFieldValues($data) {
       }
       $output[$public_field_name] = $value;
     }
+    if ($this->isCacheEnabled($data)) {
```

Comment:
I tried (unsuccessfully) to find an elegant way to do this. The problem is that we don't have only one way to generate the output, since it will depend on runtime configuration.